src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs

```diff
@@ -401,5 +414,17 @@ private NuGetPackageInfo ReadPackageInformation(string packageLocation)
                 nuspec.GetId(),
                 nuspec.GetVersion().ToNormalizedString());
         }
+
+        // This conversion is necessary as "InstallResult" does not have the support of the NuGet types
+        private Dictionary<Uri, int> ConvertVulnerabilityMetadata(IEnumerable<PackageVulnerabilityMetadata> vulnerabilities)
+        {
+            var dictionaryResult = new Dictionary<Uri, int>();
+            foreach (var vulnerability in vulnerabilities)
+            {
+                dictionaryResult[vulnerability.AdvisoryUrl] = vulnerability.Severity;
```

Comment:
does `PackageVulnerabilityMetadata` has an user-friendly description of the vulnerability?

Reply:
It contains `AdvisoryUrl` for the GitHub vulnerability database.  We could later query Github for all the details.